Exiled.API/Features/Items/Scp330.cs

```diff
@@ -58,56 +58,36 @@ internal Scp330()
         /// Adds a specific candy to the bag.
         /// </summary>
         /// <param name="type">The <see cref="CandyKindID"/> to add.</param>
-        [System.Obsolete("Please use AddCandy(CandyKindID, out bool) instead.", true)]
-        public void AddCandy(CandyKindID type)
-        {
-            AddCandy(type, out _);
-        }
-
-        /// <summary>
-        /// Adds a specific candy to the bag.
-        /// </summary>
-        /// <param name="type">The <see cref="CandyKindID"/> to add.</param>
-        /// <param name="result">Whether or not the candy was successfully added to the bag.</param>
-        public void AddCandy(CandyKindID type, out bool result)
+        /// <returns>Whether or not the candy was successfully added to the bag.</returns>
```

Comment:
`<see langword="true"/> if the candy was successfully added to the bag; otherwise, <see langword="false"/>.`
I'd use this instead, just to match .NET and C# docs returns style.